pkg/platform/aws/aws.go

```diff
@@ -158,6 +159,11 @@ func createTerraformConfigFile(cfg *config, terraformRootDir string) error {
 		return errors.Wrapf(err, "failed to marshal CLC snippets")
 	}
 
+	extraKubeAPIServerFlags, err := json.Marshal(cfg.KubeAPIServerExtraFlags)
```

Comment:
why not kubeAPIServerExtraFlags?

Reply:
Ah yes, I mixed 2 conventions while working on it, then reverted part of it.

Reply:
Fixed.